website/docs/r/key_vault_policy.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_key_vault_policy"
+sidebar_current: "docs-azurerm-resource-key-vault-policy"
+description: |-
+  Create a Key Vault Access Policy.
+---
+
+# azurerm\_key\_vault\_policy
+
+Create a Key Vault Access Policy.
+
+~> **NOTE on Key Vaults and Key Vault Policies:** Terraform currently
+provides both a standalone [Key Vault Policy Resource](key_vault_policy.html), and allows for Key Vault Access Polcies to be defined in-line within the [Key Vault Resource](key_vault.html).
+At this time you cannot define Key Vault Policy with in-line Key Vault in conjunction with any Key Vault Policy resources. Doing so may cause a conflict of rule settings and will overwrite rules.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "resourceGroup1"
+  location = "West US"
+}
+
+resource "azurerm_key_vault" "test" {
+  name                = "testvault"
+  location            = "West US"
```

Comment:
_minor_ in newer examples we're making this:

```
location = "${azurerm_resource_group.test.location}"
```

so that the user only has to update this in a single location - can we update this? :)